web/src/settings.ts

```diff
@@ -0,0 +1,2 @@
+export const API_URL = process.env.NEXT_PUBLIC_API_URL || "http://127.0.0.1:3001";
+export const STAMPY_URL = process.env.STAMPY_URL || "http://127.0.0.1:8787"
```

Comment:
this will have to be set to 'https://aisafety.info' wherever the deployed thing gets things from. CORS should work now, but only for that domain. It might be worth changing the UI to also accept localhost...